app/services/doc_auth/mock/result_response.rb

```diff
@@ -36,7 +36,7 @@ def errors
             classification_info = file_data.dig('classification_info')
 
             if [doc_auth_result, image_metrics, failed, passed, liveness_result,
-                classification_info].any?(&:present?)
+                !id_type_supported?].any?(&:present?)
```

Comment:
not sure how to check !id_type_supported? a boolean's presence?

Reply:
That's a good question...I thought it was working because when I test locally it works.

But after your comment I tried in irb - and you're right, this doesn't work.

Hmm. I'm not sure what to do. I'll have to look into this more.

Reply:
Oddly, trying again today, using the boolean was successful. If I do:

`[nil, nil, true].any?(&:present?)` it returns `true`.

`[nil, nil, false].any?(&:present?)` returns `false`.

So, that must be why it works running locally as well. But, I still think it's confusing and am trying to find a clearer way to accomplish it without changing the interface too much.

Reply:
I think an `or` statement with the boolean is a clearer way to go about this. I changed it in 9d707cf1bfde155cf3f572af4b9bf0cefd562fff. @dawei-nava please let me know what you think!